### DISCUSSION TITLE
Which max number of shard in collection is correct in Milvus 2.3 docs?

### DISCUSSION THREAD

**Discussion by dogyun-k** [milvus-io/milvus#28851] [Q&A and General discussion] [answered]
Hi team,

Milvus version : v2.3.2

The maximum number of shards listed in the two links below is different.

I wonder if two values are different or one side is wrong.

> https://milvus.io/docs/create_collection.md#Limits
> https://milvus.io/docs/limitations.md#Number-of-resources-in-a-collection

Actually, I tried to create collection with shards_num 48 and failed.

``` python
milvus_collection = Collection(
                name="my_collection",
                schema=_get_schema(),
                shards_num=48
            )
```

**Comment by yhmo** [comment, marked as answer]
16 is the correct value. Will update the doc soon.
In fact, the maximum number of shards_num is limited by the rootCoor.dmlChannelNum in the milvus.yaml:
```
rootCoord:
  dmlChannelNum: 16 # The number of dml channels created at system startup
```

**Reply by dogyun-k** [reply]
Thank you @yhmo!
And I have one more question. 
You mean if I set rootCoor.dmlChannelNum to 48, I can create collection having 48 shards, and Does it have any problem?

**Reply by yhmo** [reply]
> Thank you @yhmo! And I have one more question. You mean if I set rootCoor.dmlChannelNum to 48, I can create collection having 48 shards, and Does it have any problem?

Yes, if you set set rootCoor.dmlChannelNum to 48, the maximum number of shards_num becomes 48.
We don't recommend a high shard number because more shards will split data to more small segments, and bring higher pressure to the entire milvus service.
